sklearn/impute/_iterative.py

```diff
@@ -111,13 +111,15 @@ class IterativeImputer(_BaseImputer):
         many features with no missing values at both ``fit`` and ``transform``
         time to save compute.
 
-    min_value : float, default=None
-        Minimum possible imputed value. Default of ``None`` will set minimum
-        to negative infinity.
+    min_value : float or array-like of shape (n_features,), default=None.
+        Minimum possible imputed value. Broadcast to shape (n_features,) if
+        scalar. Expects shape (n_features,) if array-like, one min value for
```

Comment:
I would turn the sentence around `If array-like, the array should be of shape ...

Reply:
Done!